example/main.jsonnet

```diff
@@ -171,7 +166,6 @@ local dex = (import 'github.com/observatorium/observatorium/configuration/compon
       volumeClaimTemplate: obs.loki.config.volumeClaimTemplate,
     },
     securityContext: {
-      fsGroup: 65534,
```

Comment:
why do we remove this?

Reply:
ping @clyang82

Reply:
It should not be removed. In order to fix the problem `unknown field "fsGroup" in io.observatorium.core.v1alpha1.Observatorium.spec.securityContext`, I need to update api to use `PodSecurityContext`